core-bundle/src/Controller/FrontendModule/TwoFactorController.php

```diff
@@ -150,7 +150,7 @@ private function enableTwoFactor(Template $template, Request $request, FrontendU
         if ('tl_two_factor' === $request->request->get('FORM_SUBMIT')) {
             if ($authenticator->validateCode($user, $request->request->get('verify'))) {
                 // Enable 2FA
-                $user->useTwoFactor = '1';
+                $user->useTwoFactor = true;
```

Comment:
If we read the user from the database would the value not still be fetched as `'1'` (or `'0'` if false)?

Reply:
No, it will `0` or `1` (with PDO under PHP 8.1).